src/Config/RabbitMQExtensionConfigProvider.cs

```diff
@@ -152,11 +147,15 @@ public byte[] Convert(T input)
 
         internal class EventArgsToPocoConverter<T> : IConverter<BasicDeliverEventArgs, T>
         {
+            private readonly LoggerFactory _loggerFactory = new LoggerFactory();

```

Comment:
Created member variables for logging for internal class, but is there a way to pass in the logger factory/logger from the actual config provider into the converter?